### DISCUSSION TITLE
Can I use Cloud firestore Flutterfire desktop for now?

### DISCUSSION THREAD

**Discussion by huongtran84** [invertase/flutterfire_desktop#30] [Q&A] [answered]
I am going to develop my app use firebase so  Can I use Cloud firestore with Flutterfire desktop for now?

**Comment by pr-Mais** [comment, marked as answer]
You mean the Firebase REST API?
Guess so, if all you need is authentication, the progress is already going and soon it will be in a good shape, actually you can start using it from now.
But if you use others like Firestore, developing the package will take time, probably not so soon.
